WalletWasabi/Blockchain/TransactionOutputs/SmartCoin.cs

```diff
@@ -206,6 +207,12 @@ public bool Confirmed
 			private set => RaiseAndSetIfChanged(ref _confirmed, value);
 		}
 
+		public int Confirmations 
```

Comment:
Is this used somewhere? It is not updated either. In the UI it is calculated every time the TipHeight changes: https://github.com/molnard/WalletWasabi/blob/46e0d7e4d1bb6baa2bd302294a0bda4c354c5bae/WalletWasabi.Gui/Controls/WalletExplorer/CoinViewModel.cs#L152

I suggest removing, it implies the idea that this is up-to-date and can be used.

Reply:
I agree, this shouldn't be here.

Reply:
Agreed, oversight on my part as I started with a different approach initially that involved modifying SmartCoin.cs. Updated the PR to remove that field.  Thanks @molnard  and @lontivero